app/components/improvedHome/index.tsx

```diff
@@ -118,6 +120,11 @@ const ImprovedHome: React.FC = () => {
               <Icon icon="SCINAPSE_HOME_LOGO" className={styles.scinapseHomeLogo} />
             </div>
             <div className={styles.subTitle}>Academic Search Engine</div>
+            {instituteInfo ? (
```

Comment:
`{institueInfo && ()}` might be better to read. there's no alternative component for the null case